Packs/AtlassianJiraServiceManagement/Integrations/AtlassianJiraServiceManagement/AtlassianJiraServiceManagement.yml

```diff
@@ -0,0 +1,683 @@
+      description: The date in which this comment was created.
+      type: Date
+    - contextPath: JiraAsset.Comment.Updated
+      description: The date in which this object was last updated.
+      type: Date
+    - contextPath: JiraAsset.Comment.ID
+      description: The id of the newly created comment.
+      type: Number
+    - contextPath: JiraAsset.Comment.Role
+      description: ''
+      type: Number
+    - contextPath: JiraAsset.Comment.Comment
+      description: The body of the comment that was newly added.
+      type: String
+    - contextPath: JiraAsset.Comment.CommentOutput
+      description: The body of the comment that was newly added.
+      type: String
+    - contextPath: JiraAsset.Comment.ObjectId
+      description: The id of the object the new comment was added to.
+      type: Number
+    - contextPath: JiraAsset.Comment.CanEdit
+      description: Whether the comment is editable.
+      type: Boolean
+    - contextPath: JiraAsset.Comment.CanDelete
+      description: Whether the comment can be deleted.
+  - arguments:
+    - description: Id of the object that the file will be attached to.
+      name: object_id
+      required: true
+    - description: Id of the file to upload.
+      name: entry_id
+      required: true
+    name: jira-asset-attachment-add
+    description: Uploads a file attachment to the specified object.
+    outputs:
+    - contextPath: JiraAsset.Attachment.ID
+      description: Id of the attachment.
+      type: Number
+    - contextPath: JiraAsset.Attachment.Author
+      description: Author of the attachment.
+      type: String
+    - contextPath: JiraAsset.Attachment.MimeType
+      description: The file's MIME type.
+      type: String
+    - contextPath: JiraAsset.Attachment.Filename
+      description: The file's name.
+      type: String
+    - contextPath: JiraAsset.Attachment.Filesize
+      description: The size of the file.
+      type: String
+    - contextPath: JiraAsset.Attachment.Created
+      description: The date in which this attachment was created.
+      type: Date
+    - contextPath: JiraAsset.Attachment.Comment
+      description: The comment attached to the attachment, if there is one.
+      type: String
+    - contextPath: JiraAsset.Attachment.CommentOutput
+      description: The comment output of the attachment, if there is one.
+      type: String
+    - contextPath: JiraAsset.Attachment.Url
+      description: Url to the attachment.
+      type: String
+  - arguments:
+    - description: Id of the object that attachments will be fetched for.
+      name: object_id
+      required: true
+    - description: Whether to download the attachments for this object.
+      name: download_file
+      auto: PREDEFINED
+      predefined:
+      - "true"
+      - "false"
+    name: jira-asset-attachment-list
+    description: Returns a list of attachments for the specified object. You can also download the files in a zipped format.
```

Comment:
How do you download the files in a zipped format?  I don't see an argument for that.